3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/nested-arrays/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 pub fn transpose(matrix: [[i32; 3]; 3]) -> [[i32; 3]; 3] {
-    todo!("Implement this")
+    let mut t = [[0; 3]; 3];
+
+    for i in 0..3 {
+        for j in 0..3 {
+            t[j][i] = matrix[i][j];
+        }
+    }
+    return t;
```

Comment:
마지막 문장에는 `return`을 생략할 수 있습니다.
```suggestion
    t
```